osp/models/form_feedback.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.db import models
+
+from osp.models.form import Form
+from osp.models.answer import Answer
+from osp.utils import choices
+
+class FormFeedback(models.Model):
+
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+    ) 
+    form = models.ForeignKey(
+        Form,
+        on_delete=models.CASCADE
+    )
+    answers = models.ManyToManyField(
+        to=Answer,
+        default=None,
+        blank=True,
+        related_name='form_answers'
+    )
+    acceptance_status = models.CharField(
```

Comment:
acceptance status?

Reply:
Denoting the pending/accepted/rejected/waitlisted status. Just status would create confusion as in other models also we have similar details.